### ISSUE TITLE
What happens if WinAppDriver is initialized for the app that is already opened?

### ISSUE THREAD

**Issue by RamakrishnanRajaram** [microsoft/WinAppDriver#1869]
I have launched my application manually but If I tried to relaunch the same application by initializing the windows driver I could see two different things happening
1. In My machine, windows driver recognizes the application that is already opened
2. But in my colleague's machine, it is launching another instance.

`WindowsDriver<WindowsElement> windowsDriver;
			string WindowsApplicationDriverUrl = windowsApplicationDriverUrl;
			AppiumOptions appCapabilities = new AppiumOptions();
			appCapabilities.AddAdditionalCapability("app", applicationPath);
			appCapabilities.AddAdditionalCapability("appWorkingDir", applicationDirectory);
			windowsDriver = new WindowsDriver<WindowsElement>(new Uri(WindowsApplicationDriverUrl), appCapabilities);
			if(windowsDriver.Title.IsNullOrEmpty())
			{
				appCapabilities = new AppiumOptions();
				appCapabilities.AddAdditionalCapability("app", applicationPath);
				appCapabilities.AddAdditionalCapability("appWorkingDir", applicationDirectory);
				windowsDriver = new WindowsDriver<WindowsElement>(new Uri(WindowsApplicationDriverUrl), appCapabilities);  //// **This is where the two scenarios that I mentioned is happening**
				IWebElement ele = windowsDriver.FindElementByName(applicationTopLevelWindow);
				string appTopLevelWindowHex = int.Parse(ele.GetAttribute("NativeWindowHandle")).ToString("X"); 				
				appCapabilities = new AppiumOptions();
				appCapabilities.AddAdditionalCapability("appTopLevelWindow", appTopLevelWindowHex);
			}			
			return windowsDriver;`

**Comment by anunay1** [comment]
Your application supports multiple instances, better check if the application is running before you initialize the driver.

**Comment by RamakrishnanRajaram** [comment]
Yes, my application is running before I initialize my driver. But the issue is, it behaves differently in two different machines.

**Comment by anunay1** [comment]
Not an issue with win app driver, just implemented a logic to kill the application before you launch your application
